dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/MasterServer.java

```diff
@@ -191,10 +136,15 @@ public void close(String cause) {
             // close
             this.schedulerApi.close();
             this.masterSchedulerService.close();
-            this.nettyRemotingServer.close();
+            try (MasterRPCServer closeServer = this.masterRPCServer) {
```

Comment:
looks a little weird

Reply:
Ok, I remove the try, and use `this.masterRPCServer.close()`